pkg/support/dump_windows.go

```diff
@@ -18,12 +18,19 @@ package support
 
 import (
 	"flag"
+	"fmt"
 	"path"
+	"path/filepath"
+
+	"github.com/spf13/afero"
 )
 
-const antreaWindowsWellKnownLogDir = `C:\k\antrea\logs`
+const (
```

Comment:
Maybe it's better to add a TODO reminder here since we can only collect log in default path for now?

Reply:
Thanks, I will add todo here.